compiler/rustc_apfloat/src/ieee.rs

```diff
@@ -570,9 +570,9 @@ impl<S: Semantics> fmt::Display for IeeeFloat<S> {
 
             // Exponent always at least two digits if we do not truncate zeros.
             if truncate_zero {
-                write!(f, "{:+}", exp)?;
+                write!(f, "{exp:+}")?;
             } else {
-                write!(f, "{:+03}", exp)?;
```

Comment:
I think apfloat changes need to be reverted?

Reply:
not sure if that is still the case, cc @eddyb

Reply:
but it does make sense to leave this out of this PR, because it at least requires a submodule? update